backend/reviewduck/src/test/java/com/reviewduck/template/controller/TemplateControllerTest.java

```diff
@@ -94,7 +94,17 @@ void nullQuestionDescription(String description) throws Exception {
             // when, then
             assertBadRequestFromPost("/api/templates", request, "템플릿의 질문 설명 생성 중 오류가 발생했습니다.");
         }
+    }
+
+    @Nested
+    @DisplayName("템플릿 검색 결과 조회")
+    class SearchTest {
 
+        @Test
+        @DisplayName("query 파라미터에 값이 없을 경우 예외가 발생한다.")
```

Comment:
query 파라미터에 값이 없을 경우가 모호한 것 같아요. 

1. query라는 param 자체가 존재하지 않는 경우
2. query param에 해당하는 값이 존재하지 않는 경우 

```suggestion
        @DisplayName("query 파라미터 키를 포함하지 않을 경우 예외가 발생한다.")
```
이 정도가 떠오르는데, 더 좋은 의견이 있으시면 적용해주시면 좋을 것 같습니다👍

Reply:
세분화하여서 테스트 분리하였습니다!